src/SFML/Window/Unix/WindowImplX11.cpp

```diff
@@ -367,6 +369,35 @@ bool isWMAbsolutePositionGood()
                        std::end(wmAbsPosGood),
                        [&](const sf::String& name) { return name == windowManagerName; });
 }
+
+// Initialize raw mouse input
+void initRawMouse(::Display* disp)
+{
+    auto opcode = 0;
+    auto event  = 0;
+    auto error  = 0;
+
+    if (XQueryExtension(disp, "XInputExtension", &opcode, &event, &error))
+    {
+        auto major = 2;
+        auto minor = 0;
```

Comment:
```suggestion
        int major = 2;
        int minor = 0;
```

Also, can these be `const`?

Reply:
`XIQueryVersion` takes their pointers as in/out parameters, so they can't be `const`.